Configuration/Icons.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+    // Icon identifier
+    'tea_icon' => [
+        // Icon provider class
+        'provider' => \TYPO3\CMS\Core\Imaging\IconProvider\SvgIconProvider::class,
+        // The source SVG for the SvgIconProvider
+        'source' => 'EXT:tea/Resources/Public/Icons/Extension.svg',
```

Comment:
Please let's use a dedicated file for this (even if we copy the extension icon). My proposal for a name is `Plugin.svg`.

Reply:
Done, i also used the same one for the plugin registration